guides/v2.3/cloud/live/go-live-checklist.md

```diff
@@ -91,14 +75,14 @@ Test and verify Fastly caching is correctly working in Production. For detailed
 
 ## Performance testing {#performance}
 
-We recommend that you review the [Magento Performance Toolkit]({{ site.mage2300url }}setup/performance-toolkit) options as part of your pre-launch readiness process.
+We recommend that you review the [Magento Performance Toolkit]({{ site.mage2200url }}setup/performance-toolkit) options as part of your pre-launch readiness process.
```

Comment:
just to be sure that there is no mistake:
1) here you give a link from 2.3 documentation to 2.2 documentation 
2) the link doesn't seem to be working. It should end with `/` if there is an index.html file in the `setup/performance-toolkit` directory, or `.html` if `performance-toolkit` is an HTML file

Reply:
this link goes to a GitHub repo -- it works in the published version.  However, you are correct that it should point to the 2.3 branch of the repo on this topic.   -- Good catch.

I updated the link to point to the right branch and built locally-- it opens to the correct branch of the repo: https://github.com/magento/magento2/tree/2.3/setup/performance-toolkit

Reply:
sorry, just noticed my mistake about the variable